public/data/latenighthunter.json

```diff
@@ -1,12 +1,12 @@
 {
-  "name": "Giovanny Martinez",
+  "name": "Latenighthunter",
   "type": "personal",
   "bio": "Open Source Enthusiast!",
-  "avatar": "https://github.com/giovanny-martinez.png",
+  "avatar": "https://github.com/Latenighthunter.png",
   "links": [
     {
       "name": "Follow me on GitHub",
-      "url": "https://github.com/giovanny-martinez",
+      "url": "https://github.com/Latenighthunter",
       "icon": "github"
     },
     {
```

Comment:
I think that adding the link to your profile in your profile is a little bit redundant (?) Perhaps you would like to post this link somewhere else (your other profiles for example) so that it leads people to this LinkFree profile instead.



__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix